src/ServiceControl/Hosting/Commands/ImportFailedAuditsCommand.cs

```diff
@@ -31,7 +31,7 @@ void RunAndWait(HostArguments args)
 
             var loggingSettings = new LoggingSettings(settings.ServiceName, LogLevel.Info, LogLevel.Info);
             var bootstrapper = new Bootstrapper(() => { tokenSource.Cancel();}, settings, busConfiguration, loggingSettings);
-            var bus = (UnicastBus)bootstrapper.Start();
+            var bus = (UnicastBus)bootstrapper.Start().Bus;
```

Comment:
This is only needed for resolving. Can we do this in another way without exposing the bus?

Reply:
We can't expose the `IBuilder` but we can expose the underlying Autofac container instead of `IBus`. What do you think?

Reply:
I would prefer to expose the container but I still think it is leaky. Normally the bootstrapper should know what to bootstrap. I get that this is basically a non-normal bootstrapping when we want to import the failed audits. That's why we are awaiting here the import audits. Still, it seems weird that we start the bootstrapper and then need to resolve the audit import stuff.

Reply:
Yeah. This is an edge case but I agree it is ugly. The proper way of building this feature would have been an admin UI that allows to start/stop the importing of failed audits. We wanted to cut corners here :-(